contrib/babelfishpg_common/Makefile

```diff
@@ -78,6 +78,7 @@ include $(PGXS)
 
 ifeq ($(GE91),yes)
 all: sql/$(EXTENSION)--$(EXTVERSION).sql $(UPGRADES)
+all: sql/upgrades/$(EXTENSION)--3.2.0--3.3.0.sql sql/upgrades/babelfishpg_upgrades.in
```

Comment:
So far, we don't have other upgrade files, e.g., `3.1.0--3.2.0.sql`. Why do we need this?

Reply:
Actually, for upgrade files now we need to add both changes related to spatial datatypes and other datatypes, but if we don't enable the spatial Datatype switch then we don't want these changes in upgrade files also so that's why we have created 3 extra files - spatial_types--3.2.0--3.3.0.sql,babelfish_common_helper--3.2.0--3.3.0.sql, babelfishpg_upgrades.in, so that we can automate upgrade file creation through this MakeFile code, and there's no risk of breaking old upgrade test/paths, this is a seperate section which will not affect the old code, earlier we have to manually create upgrade files, e.g. ```3.1.0--3.2.0.sql```